karyon3-core/src/main/java/com/netflix/karyon/Karyon.java

```diff
@@ -283,83 +320,90 @@ public LifecycleInjector start() {
     }
     
     /**
-     * Shortcut to creating the injector
-     * @return The builder
      */
     public LifecycleInjector start(String[] args) {
-        if (this.getPropertySource().equals(DefaultPropertySource.INSTANCE) && isFeatureEnabled(KaryonFeatures.USE_ARCHAIUS)) { 
-            try {
-                apply((KaryonModule) Class.forName("com.netflix.karyon.archaius.ArchaiusKaryonModule").newInstance());
-            }
-            catch (ClassNotFoundException e) {
-                throw new RuntimeException("Unable to bootstrap using archaius. Either add a dependency on 'com.netflix.karyon:karyon3-archaius2' or disable the feature KaryonFeatures.USE_ARCHAIUS");
-            } 
-            catch (InstantiationException | IllegalAccessException e) {
-                throw new RuntimeException("Unable to bootstrap using archaius");
-            }
+        for (ModuleListTransformer transformer : transformers) {
+            modules = transformer.transform(Collections.unmodifiableList(modules));
         }
         
-        String karyonProfiles = getPropertySource().get(KARYON_PROFILES);
-        if (karyonProfiles != null) {
-             addProfiles(karyonProfiles);
-        }
+        this.addAutoBinder(TypeLiteralMatchers.subclassOf(PropertySource.class), new PropertySourceAutoBinder());
```

Comment:
Can I bind directly PropertySource? I assume this is a default applied when none was provided?